src/Util.java

```diff
@@ -0,0 +1,16 @@
+public class Util {
+
+    public static int isInteger( String input, Player player ) {
```

Comment:
The function name does not make sense. You should use the name that convey the process.